src/manual/design/devel/idempotency.md

```diff
@@ -0,0 +1,33 @@
+---
+filename: design/devel/idempotency.md
+title: Idempotency
+order: 30
+---
+
+All Taskcluster API operations are idempotent. This means that repeating a
+request has no effect, as long as it occurs within a reasonable time.
```

Comment:
I think this is a bold statement.  This is not an inherent property of our API framework and has to explicitly be supported in a lot of cases.  I agree that all *should*, but this statement does not sit well with me.